modules/release-notes/pages/relnotes.adoc

```diff
@@ -1,10 +1,306 @@
+
+|===
+
+==== Data Service
+
+[#table_fixedissues_v702-data,cols="25,66"]
+|===
+|Issue | Description
+
+
+| https://issues.couchbase.com/browse/MB-38978[MB-38978^]
+| *Summary*: STAT "dcp" and "dcpagg" adversely affect front-end operation latency
+
+| https://issues.couchbase.com/browse/MB-47267[MB-47267^]
+| *Summary*: Vbucket stats call to KV can timeout during delta node recovery preparation
+
+|===
+
+==== Query Service
+
+[#table_fixedissues_v702-query,cols="25,66"]
+|===
+|Issue | Description
+
+
+| https://issues.couchbase.com/browse/MB-46876[MB-46876^]
+| *Summary*: AT_PLUS queries with collections are not working
+
+|===
+
+==== Index Service
+
+[#table_fixedissues_v702-gsi,cols="25,66"]
+|===
+|Issue | Description
+
+
+| https://issues.couchbase.com/browse/MB-46725[MB-46725^]
+| *Summary*: Rebalance button not enabled post Quorum Loss failover even when indexing has partitioned indexes
+
+| https://issues.couchbase.com/browse/MB-46350[MB-46350^]
+| *Summary*: UI shows 104k mutations remaining when creating indexes on empty bucket
+
+| https://issues.couchbase.com/browse/MB-47047[MB-47047^]
+| *Summary*: Internal Server error is raised while performing backup on a index node using cbbackupmgr
+
+| https://issues.couchbase.com/browse/MB-47631[MB-47631^]
+| *Summary*: num_rollbacks_to_zero stats not changing for the rollback to zero
+
+| https://issues.couchbase.com/browse/MB-47635[MB-47635^]
+| *Summary*: Optimise cluster info cache access in metadata_provider and request_handler
+
+| https://issues.couchbase.com/browse/MB-47684[MB-47684^]
+| *Summary*: num_rollbacks_to_zero stats not changing for the rollback to zero
+
+| https://issues.couchbase.com/browse/MB-47737[MB-47737^]
+| *Summary*: Log more information when replica counter merge fails
+
+| https://issues.couchbase.com/browse/MB-47760[MB-47760^]
+| *Summary*: handleIndexMergeSnapshot() panic from extraneous s.muSnap.Unlock() calls
+
+| https://issues.couchbase.com/browse/MB-47833[MB-47833^]
+| *Summary*: getIndexStatus endpoint is getting timed out
+
+| https://issues.couchbase.com/browse/MB-47878[MB-47878^]
+| *Summary*: CBSE - Increase the poll time in pollForDeletedBuckets at projector
+
+| https://issues.couchbase.com/browse/MB-48017[MB-48017^]
+| *Summary*: CPU utilization is much higher in 7.0.1/2 when cluster is idle
+
+| https://issues.couchbase.com/browse/MB-48030[MB-48030^]
+| *Summary*: [TLS] Strict mode, see errors in query log for 9101 port for indexer client
+
+| https://issues.couchbase.com/browse/MB-48144[MB-48144^]
+| *Summary*: Add version check to enable plasma indexes on Ephemeral buckets from 7.0.2
+
+| https://issues.couchbase.com/browse/MB-48145[MB-48145^]
+| *Summary*: [Enforce-TLS] IndexStatus returns Cannot communicate with indexer process. Information on indexes may be stale. Will retry
+
+| https://issues.couchbase.com/browse/MB-48146[MB-48146^]
+| *Summary*: [IPV6] no index available when switching to ipv6 or ipv6only IP family
```

Comment:
Remove this one, ipv6only didn't exist before this release.

Reply:
Removed.